### PR TITLE
add underscores in filename validation

### PR DESCRIPTION
**Pull request by MbfloydIR**
## Summary
https://tsu.atlassian.net/browse/IR-4621
## References
closes #_insert number here_

### CODE REVIEW

File: packages/common/tests/regex.test.ts [hunk 1]
```diff
@@ -48,7 +48,6 @@ describe('regex.test', () => {
   describe('INVALID_FILENAME_REGEX', () => {
     it('should not match invalid filenames', () => {
       const invalidFilenames = [
-        'hello_world',
```
Comment (aditya-mitra) [inline packages/common/tests/regex.test.ts:51]: can you also a add a test with `_` in the _should match valid filenames_ test?

**Review by aditya-mitra — DISMISSED** [review]

File: packages/common/tests/regex.test.ts (hunk 1 above)
Comment (Rezmason) [reply packages/common/tests/regex.test.ts:51]: I'll do it 😎

Comment (Rezmason) [reply packages/common/tests/regex.test.ts:51]: Added https://github.com/ir-engine/ir-engine/pull/349/commits/32225efcc4dbdb579a6a231601621ab744a59733

Comment (Rezmason) [reply packages/common/tests/regex.test.ts:51]: Apparently `_` should not be a valid filename. Also, `example.com` should also be invalid, but it isn't. Investigating.

Comment (Rezmason) [reply packages/common/tests/regex.test.ts:51]: with insight offered by David and Kyle, we’ll support the following conditions for filenames.

A filename is valid if:
- it has 4 to 64 characters
- its first and last character are alphanumeric
- any other characters are either alphanumeric, “_”, “-”, or “.”

**Review by barankyle — APPROVED** [review]

**Pull request merged**